wikipron/extract/__init__.py

```diff
@@ -0,0 +1,10 @@
+from wikipron.extract.default import extract_word_pron_default
+from wikipron.extract.khm import extract_word_pron_khmer
+
+
+# All extraction functions must have the exact same function signature.
+EXTRACTION_FUNCTIONS = {
+    "default": extract_word_pron_default,
+    # Key has to be the language name used by Wiktionary.
+    "Khmer": extract_word_pron_khmer,
+}
```

Comment:
Here's the high-level idea: For every language we need special support for, define its own "extract_word_pron_X" function and add it here.

Reply:
+1